integration-tests/deployment/ccip/consts.go

```diff
@@ -0,0 +1,9 @@
+package ccipdeployment
+
+type TokenDescriptor string
+
+const (
+	LinkDescriptor TokenDescriptor = "LINK"
+	WETHDescriptor TokenDescriptor = "WETH"
```

Comment:
Do you mean to call this `Symbol`? Descriptor is typically used as a "handle to" something (e.g file descriptor).

Reply:
I'm okay with `Symbol` as well 👍